StackCalculator/StackCalculatorProj/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using StackCalculatorProj;
+using StackCalculatorProj.Stacks.Impl;
+
+
+Console.WriteLine("Insert Reverse Polish Notation type string to calculate");
```

Comment:
```suggestion
using StackCalculatorProj.Stacks.Impl;

Console.WriteLine("Insert Reverse Polish Notation type string to calculate");
```
Ну и скорее "Input", "Insert" про ввод данных не говорят.